ow_sim_tests/scripts/test_arm_find_surface.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import sys
+import rospy
+import roslib
+import unittest
+import ow_lander.msg
+import owl_msgs.msg
+import argparse
```

Comment:
The vscode will make the fonts a little darker if the import not use, so it easy fix for me. Also, I find out one thing, since we do 'from action_testing import * ' do you think it also is nice if I clean some of the duplicate import? For example import rospy is not necessary for all the unit test files because when we do 'from action_testing import * ' it will do the 'import rospy'

Reply:
> The vscode will make the fonts a little darker if the import not use, so it easy fix for me. Also, I find out one thing, since we do 'from action_testing import * ' do you think it also is nice if I clean some of the duplicate import? For example import rospy is not necessary for all the unit test files because when we do 'from action_testing import * ' it will do the 'import rospy'

Hmmm, that's a good point. I would rather we keep `from action_testing import *` and the redundant imports for now. Using `*` in import statements is actually discouraged in the Python world, but I use it here simply to make lines shorter. When I make it a more explicit import statement, _e.g._ `import action_testing` then suddenly the files will require those other import statements again. The small refactor of `action_testing.py` that I have in mind will use less global scope, so I plan to make it an explicit import then. 

So to summarize, please keep the redundant imports for now.